### ISSUE TITLE
Allow organization-based boards

### ISSUE THREAD

**Issue by froboy** [mglaman/contribkanban.com#116]
I have no idea if this would be possible with the existing data from d.o, but it'd be really interesting to have build a board that contains all of the issues that have participation from a particular organization. This could help orgs and teams better manage their contrib work and make sure issues aren't falling through the cracks.

**Comment by mglaman** [comment]
I don't think this can be tracked via the API. One solution is to build a Node Board. It can be semi-private and shared via a link, or public so that it is collaborative. The interface needs some love (like pruning closed/old issues)

<img width="1118" alt="screen shot 2019-01-29 at 9 16 33 pm" src="https://user-images.githubusercontent.com/3698644/51955885-2bd30280-240b-11e9-9de2-eb7f9c0e588e.png">

**Comment by deviantintegral** [comment]
This is possible as long as the organization is listed as a sponsor of the project on the project node. If you start from the organization node, you can relate into projects, and then into issues.
